core/domain/platform_parameter_registry.py

```diff
@@ -213,8 +212,6 @@ def update_platform_parameter(
         # Create a temporary param instance with new rules for validation,
         # if the new rules are invalid, an exception will be raised in
         # validate() method.
```

Comment:
Just to check, this will validate that the default value has the correct type for the given platform parameter?

Reply:
Yes, that is correct.